packages/elemental-theme/src/custom-elements/ef-combo-box.less

```diff
@@ -19,7 +21,11 @@
   }
 
   [part=input] {
```

Comment:
[part~=input]

Reply:
Is this for future support? Because we're currently set `part="input"` for now.

Reply:
Yes, we should always use `~=` to be closer to part css syntax